core/src/main/java/org/apache/calcite/sql/type/SqlTypeName.java

```diff
@@ -151,7 +153,7 @@
   public static final List<SqlTypeName> ALL_TYPES =
       ImmutableList.of(
           BOOLEAN, INTEGER, VARCHAR, DATE, TIME, TIMESTAMP, NULL, DECIMAL,
-          ANY, CHAR, BINARY, VARBINARY, VARBYTE, BYTE, TINYINT, SMALLINT, BIGINT, BYTEINT, REAL,
+          ANY, CHAR, BINARY, VARBINARY, VARBYTE, BYTE, BLOB, TINYINT, SMALLINT, BIGINT, BYTEINT, REAL,
```

Comment:
nit: line is longer than 100 chars, looks like stylecheck tests are failing because of this.